backend/app/models/open_call.rb

```diff
@@ -0,0 +1,22 @@
+class OpenCall < ApplicationRecord
+  extend FriendlyId
+  friendly_id :investor_prefixed_name, use: :slugged
+
+  belongs_to :investor
+
+  validates :instrument_type, inclusion: {in: InstrumentType::TYPES, allow_blank: true}, presence: true
+  validates :ticket_size, inclusion: {in: TicketSize::TYPES, allow_blank: true}, presence: true
+  validates :sdgs, array_inclusion: {in: Sdg::TYPES}
+
+  validates_presence_of :closing_at, :language
+
+  translates :name, :description, :money_distribution, :impact_description
```

Comment:
First time to use Traco and even though I like implementation part of gem, to have column per language at same table is kind of violation of data normalization.

Reply:
We used globalize for example in other projects. Huge pain in the ass with all of those joins and complex queries.

Reply:
Yup, I see your point ;). Liked flexibility of joins (super easy to add new language without migrations) and had a few (probably just two) simple methods which heavy lifted all complexity. On other hand, I understand that it will be probably slightly slower -- would love to see some comparison, because foreigner keys are indexed and tables are small.
 - with joins, you get one row with all data per language at db --> and do filtering at db which language you are interested in
 - without joins, you load all data for all languages to app and do filtering there

Whatever, your points are absolutely valid ;)